src/ledger/parse/utils.ts

```diff
@@ -112,6 +114,7 @@ function parseOutcome(tx: any): any|undefined {
     fee: common.dropsToXrp(tx.Fee),
     balanceChanges: balanceChanges,
     orderbookChanges: orderbookChanges,
+    channel: channel,
```

Comment:
@adrianhopebailie How about calling this `channelChanges` for consistency with `orderbookChanges`?